src/main/views/privacy-policy.njk

```diff
@@ -0,0 +1,125 @@
+          { cookieUrl: '/cookies'}) | safe }}
+        </li>
+      </uL>
+
+      <h2 class="heading-medium">{{ t('What we do with your data') }}</h2>
+
+      <p>
+        {{ t('HMCTS is an executive agency of the Ministry of Justice and their '
+        + '<a href="{{ personalInformationCharter }}">personal information charter</a> explains how we process data.',
+        { personalInformationCharter: 'https://www.gov.uk/government/organisations/home-office/about/personal-information-charter'}) | safe }}
+      </p>
+
+      <p>
+        {{ t('Your personal data is processed by our staff in the UK, and the data is stored in the UK.') }}
+      </p>
+      <p>
+        {{ t('We use GOV.UK Notify to send emails.
+        These are processed within the EEA until the point where emails are handed over to your email provider.') }}
+      </p>
+
+      <h2 class="heading-medium">{{ t('How long we keep your data') }}</h2>
+
+      <p>{{ t('Your data is stored for one month and is then deleted.') }}</p>
+
+      <h2 class="heading-medium">{{ t('What we would also like to do with your data') }}</h2>
+
+      <p>
+        {{ t('As part of your claim we’ll ask if you want to receive notifications to let you know how '
+        + 'your application is proceeding. We’ll ask for your name and preferred contact details.') }}
+      </p>
+
+      <p>
+        {{ t('This information is not shared with third parties and you can unsubscribe at any time '
+        + 'by phoning 0203 123 1234 or emailing mydata@hmcts.net.') }}
+      </p>
+
+      <h2 class="heading-medium">{{ t('Your rights') }}</h2>
+
+      <p> {{ t('You can ask:') }}</p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('to see the personal data that we hold on you') }}</li>
+        <li>{{ t('to have the personal data corrected, removed or deleted') }}</li>
+        <li>{{ t('that access to this data is restricted (for example made temporarily unavailable)') }}</li>
+        <li>{{ t('that we hold your data for longer') }}</li>
+      </ul>
+
+      <h2 class="heading-medium">{{ t('Disclaimer') }}</h2>
+
+      <p>
+        {{ t('While we make every effort to keep GOV.UK up to date, we don’t provide any guarantees,
+        conditions or warranties that the information will be:') }}
+      </p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('current') }}</li>
+        <li>{{ t('secure') }}</li>
+        <li>{{ t('accurate') }}</li>
+        <li>{{ t('complete') }}</li>
+        <li>{{ t('free from bugs or viruses') }}</li>
+      </ul>
+
+      <p> {{ t('Email mydata@hmcts.net if you want to see any of this information.') }}</p>
+
+      <h2 class="heading-medium">{{ t('How to complain') }}</h2>
+
+      <p> {{ t('If you want to complain about how we have handled your personal data email mydata@hmcts.net.') }}</p>
+
+      <p>
+        {{ t('You can complain to the '
+        + '<a href="{{ informationCommissionerOffice }}">Information Commissioner’s Office </a> if you: ',
```

Comment:
The same here, I would suggest simple `uri`.

Reply:
👍